src/components/Subscription/Subscription.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { loadStripe } from '@stripe/stripe-js';
+import RaisedButton from 'material-ui/RaisedButton';
+import constants from '../constants';
+import { IconSteam } from '../Icons';
+
+import { useStrings } from '../../hooks/useStrings.hook';
+
+const stripePromise = process.env.REACT_APP_STRIPE_PUBLIC_KEY
+  ? loadStripe(process.env.REACT_APP_STRIPE_PUBLIC_KEY)
+  : null;
+
+const PageContainer = styled.div`
+  width: 80%;
+  margin: 0 auto;
+
+  @media only screen and (max-width: 768px) {
+    width: 100%;
+  }
+
+  & li {
+    list-style-type: initial;
+  }
+
+  h1,
+  h2 {
+    text-align: center;
+  }
+
+  & h2 {
+    font-size: 1.17em;
+  }
+`;
+
+const SubContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const SubLeft = styled.div`
+  margin: 8px;
+`;
+
+const SubRight = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const handleSubscribe = async (user) => {
+  if (!stripePromise) {
+    console.warn('Stripe integration is not configured, cannot subscribe');
+    return;
+  }
+  const stripe = await stripePromise;
+  const result = await stripe?.redirectToCheckout({
+    lineItems: [
+      {
+        price:
+          process.env.NODE_ENV === 'development'
+            ? 'price_1LE6FHCHN72mG1oK4E4NdERI'
+            : 'price_1LE5NqCHN72mG1oKg2Y9pqXb',
+        quantity: 1,
+      },
+    ],
+    mode: 'subscription',
+    successUrl: `${process.env.REACT_APP_API_HOST}/api/subscribeSuccess`,
+    cancelUrl: window.location.href,
+    clientReferenceId: user.account_id,
+  });
+  // If `redirectToCheckout` fails due to a browser or network
+  // error, display the localized error message to your customer
+  // using `error.message`.
+  if (result && result.error) {
+    console.error(result.error.message);
+  }
+};
+
+const Subsciption = ({ user, isSubscribed }) => {
+  user = true;
+  const strings = useStrings();
+  const handleManage = useCallback(async () => {
+    const res = await fetch(`${process.env.REACT_APP_API_HOST}/api/manageSub`, {
+      credentials: 'include',
+      method: 'POST',
```

Comment:
I think you have to include a return_url here  in the body? refer to watchparty manageSub call for specifics